src/components/bluebutton.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './bluebutton.scss';
+
+const Button = ({ targetTo, children, label, ...props }) =>
+(
+  <button
+    className={styles.Button}
+    tabIndex="-1"
+    {...props}
+    onClick={() => { targetTo(); }}
```

Comment:
You can remove the `;`.  If you do that, you can also remove the `{` and `}` arround `targetTo()`.